test/tests/userobjects/property_read_file/tests

```diff
@@ -0,0 +1,32 @@
+[Tests]
+  issues = '#28190'
+  design = 'PropertyReadFile.md'
+  # See test/tests/functions/piecewise_constant_from_csv for additional tests
```

Comment:
Thanks for adding this comment, I would be otherwise confused as to where the previous tests are.